oic.r.csr.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 0.8
+title: OICSecurityCsrResource
+version: v1.0-20150819
+schemas:
+  - Csr: !include schemas/oic.r.csr.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/csr:
+  description: |
+    This resource specifies a Certificate Signing Request.
+
+  displayName: Certificate Signing Request
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves CSR data.
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Csr
+            example: |
+              {
+              "rt": ["oic.r.csr"],
+              "encoding" : "oic.sec.encoding.pem"
+              "csr": "PEMENCODEDCSR"
+              }
+      503:
```

Comment:
The "sct" (supported credentials type) property of the doxm resource has a 1 bit set in the correct position (I think it's the 0x8 bit).